docs/config/01-configuration-file.md

```diff
@@ -793,6 +793,20 @@ If set then the following fields will be defined and can be overriden:
 All of Karma's urls get prefixed with the `urlRoot`. This is helpful when using proxies, as
 sometimes you might want to proxy a url that is already taken by Karma.
 
+## browserSocketTimeout
+**Type:** Number
+
+**Default:** `2000`
```

Comment:
The default provided upstream in socket.io is 20000.  Is there any reason we should not use this default as well?  

To me, at worst the extra time will mean that, should the karma server die after launching the browser, the test will last another 18secs. That seems like a small cost for an unusual case.

Reply:
I changed the default to 20000 in PR #3103 so just change the value here.

Reply:
Done!